config/locales/blacklight.en.yml

```diff
@@ -1,6 +1,6 @@
 en:
   blacklight:
-    application_name: 'CHF Hydra'
+    application_name: 'Digital Collections'
```

Comment:
Do we maybe want this to say "CHF Digital Collections"? It shows up in places like email subjects and stuff, as well as HTML `<titles>` (and from that: bookmarks, facebook shares, etc). I suppose one could even argue for "Chemical Heritage Foundation Digital Collections", I dunno.